docs/pact-reference.md

```diff
@@ -1343,7 +1344,7 @@ or to symbols imported into the namespace with [use](#use).
 ### S-expressions {#sexp}
 
 S-expressions are formed with parentheses, with the first atom determining if
-the expression is a [special form](#special) or a function application, in
+the expression is a [special form](#special-forms) or a function application, in
```

Comment:
bless you

Reply:
I clicked every. damn. one. :sweat_smile: